### PR TITLE
Disable frame-stable playback in the editor

### LINKED ISSUE #5648: Seeking in editor is laggy
**Describe the bug:** 
Map seeking is very laggy in the editor. 

**Screenshots or videos showing encountered issue:** 
https://streamable.com/0my1e

**osu!lazer version:** 
2019.807.0.0

**Logs:** 
[runtime.log](https://github.com/ppy/osu/files/3482264/runtime.log)
[performance.log](https://github.com/ppy/osu/files/3482266/performance.log)
[network.log](https://github.com/ppy/osu/files/3482267/network.log)
[database.log](https://github.com/ppy/osu/files/3482268/database.log)

### PR DESCRIPTION
**Pull request by smoogipoo**
Resolves #5648

### CODE REVIEW

File: osu.Game/Rulesets/UI/FrameStabilityContainer.cs [hunk 1]
```diff
@@ -113,6 +118,9 @@ private void updateClock()
 
             try
             {
+                if (!FrameStablePlayback)
```
Comment (peppy) [inline osu.Game/Rulesets/UI/FrameStabilityContainer.cs]: any reason this couldn't be combined with the `if (!isAttached)` below?

**Comment by smoogipoo** [comment]
Your change to this branch has regressed the behaviour. The `newProposedTime` path is exactly the path that needs to be avoided.

**Comment by peppy** [comment]
Not sure I follow, does it have something to do with `firstConsumption`? I think that should only happen once, no?

**Comment by smoogipoo** [comment]
No, it has to do with the 60fps limiting.

**Comment by smoogipoo** [comment]
Seems ok now.

**Pull request merged**